superx/information_extractor.py

```diff
@@ -0,0 +1,201 @@
+import requests
+from bs4 import BeautifulSoup
+from datetime import datetime
+import gzip
+import xml.etree.ElementTree as ET
+from superx.models.__init__ import Product, BranchPrice
+from superx.app import db
+
+
+class info_extractor:
+    """
```

Comment:
python naming convention suggests that class names should use CapWords convention. The class name would be InfoExtractor
https://www.python.org/dev/peps/pep-0008/#class-names